addon/components/rdf-input-fields/currency-input.js

```diff
@@ -9,12 +11,30 @@ export default class RdfInputFieldsCurrencyInputComponent extends SimpleInputFie
 
   @action
   updateValue(e) {
+    let updatedValue = null;
     if (e && typeof e.preventDefault === 'function') e.preventDefault();
 
     if (e.target && e.target.value) {
       this.value = e.target.inputmask.unmaskedvalue();
+      updatedValue = literal(Number(this.value), this.datatype);
```

Comment:
Should we just return the value as an integer always? `inputmask.unmaskedValue` should always return a number here ( https://robinherbots.github.io/Inputmask/#/documentation/numeric  and  ctrl+f  "unmaskAsNumber"). Than we can remove the `get datatype`

Reply:
Always a decimal you mean?

Reply:
Yes a decimal indeed
![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/7ce1822f-c9b4-4838-b472-02e1bb594123)

But with that in mind we cannot do do it because a decimal is not an integer 😄